GiniVision/Classes/Core/Logger.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Logger.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 5/14/18.
+//
+
+import Foundation
+
+final class Logger {
+    
+    enum Event {
+        case error
+        case success
+        case warning
+        case custom(emoji: String)
+        
+        var value: String {
+            switch self {
+            case .error: return "❌"
+            case .success: return "✅"
+            case .warning: return "⚠️"
+            case .custom(let emoji): return emoji
+            }
+        }
+    }
+    
+    class func debug(message: String,
```

Comment:
I don't see a point in having a debug method if there is no other log level. I'm not saying that we need log levels (but I'm also not saying that we shouldn't have them).

The point is that if you don't want logging levels, just have a generic `log` method.

Reply:
Yes, makes sense since we are not going to introduce more log levels.